app/models/bank_account.rb

```diff
@@ -46,6 +47,9 @@ def duplicates
   end
 
   def hash_data
+    # WIP because we need to get everything written onto the _routing_number version before we can proceed.
+    # Then, we will remove the attr_encrypted and change the name of the column from _routing_number to routing_number
+    self._routing_number = routing_number if routing_number_changed?
```

Comment:
Not a blocker, but I would find it easier to read if this were its own method e.g. `before_save :dual_write_routing_number, :hash_data` since this isn't strictly related to hashing the data.